src/ItemNote.ts

```diff
@@ -212,6 +217,11 @@ const generateInitialItemNoteContents = (
     ["tags", (item) => hashtagSubstitutor(true)(item.tags)],
     ["tags-no-hash", (item) => hashtagSubstitutor(false)(item.tags)],
     ["pocket-url", (item) => getPocketItemPocketURL(item)],
+    ["status", (item) => String(item.status) ?? "no status"],
```

Comment:
Please make the status be translated to a string, i.e. "unread", "archived" or "deleted". It's more user-friendly, and otherwise the status would be a number if it exists, and a string "no status" if it does not exist.

Reply:
That's a good idea, will do that.